.travis/run.sh

```diff
@@ -2,8 +2,10 @@
 set -e
 set -x
 
-if [[ "$(uname -s)" == "Darwin" ]]; then
-    # Initialize the virtualenv created at install time.
```

Comment:
For pypy we also need to source the venv...

so instead of adding a condition for the OS / Travis ENV, I went for sourcing anything which was created at the install time.

In this way, if later we add a PYPY3 for example in install.sh we don't need to add a new source condition here in run.sh